plugins/woocommerce/tests/e2e-pw/tests/shopper/account-email-receiving.spec.js

```diff
@@ -1,20 +1,36 @@
-const { test, expect } = require( '@playwright/test' );
+const { test: baseTest, expect } = require( '../../fixtures/fixtures' );
+const { customer } = require( '../../test-data/data' );
 
-const email = `test-${ Math.random() }@example.com`;
-const username = `newcustomer-${ Math.random() }`;
 const emailContent = '#wp-mail-logging-modal-content-body-content';
-const emailContentJson = '#wp-mail-logging-modal-format-json';
+const emailContentHtml = '#wp-mail-logging-modal-format-html';
+
+const test = baseTest.extend( {
+	storageState: process.env.ADMINSTATE,
+	user: async ( { api }, use ) => {
+		const now = Date.now();
+		const user = {
+			username: `${ now }`,
+			email: `${ now }@example.com`,
+			firstName: `Customer`,
+			lastName: `the ${ now }th`,
+			role: 'Customer',
+		};
+		await use( user );
+		console.log( `Deleting user ${ user.id }` );
+		await api.delete( `customers/${ user.id }`, { force: true } );
+	},
+} );
 
 test.describe(
 	'Shopper Account Email Receiving',
 	{ tag: [ '@payments', '@services' ] },
 	() => {
 		test.use( { storageState: process.env.ADMINSTATE } );
```

Comment:
This `test.use` is redundant, it's already set above. The same with the one in the next describe block.

Reply:
Ah, that's some leftover from the past. Removed.